config/settings_data.json

```diff
@@ -538,139 +538,158 @@
       ]
     }
   },
-  "features": {
-    "OrdersBrowsing": {
+  "features": [
+    {
+      "Name": "OrdersBrowsing",
       "IsActive": true,
-      "Behavior": {
-        "Requires": ["PaymentBrowsing"],
-        "Replaces": []
-      }
+      "Requires": [],
+      "Replaces": [],
+      "Replaced": [],
```

Comment:
Why are these two parameters necessary? Is this not redundant?

Reply:
Agree. The replaced property was removed